RecoEgamma/EgammaTools/python/egammaObjectModificationsInMiniAOD_cff.py

```diff
@@ -59,3 +59,68 @@ def setup_mva(val_pset,cat_pset,prod_name,mva_name):
           egamma_modifications[3].photon_config,
           pho_mva_prod_name,
           pho_spring15_50_nt.mvaSpring15NonTrigClassName+pho_spring15_50_nt.mvaTag)
+
+#############################################################
+# REGRESSION MODIFIERS
+#############################################################
+
+egamma_modifications.append(
+    cms.PSet( modifierName    = cms.string('EGExtraInfoModifierFromDB'),  
+              autoDetectBunchSpacing = cms.bool(True),
+              bunchSpacingTag = cms.InputTag("addPileupInfo:bunchSpacing"),
+              manualBunchSpacing = cms.int32(50),              
+              rhoCollection = cms.InputTag("fixedGridRhoFastjetAll"),
+              vertexCollection = cms.InputTag("offlinePrimaryVertices"),
+              electron_config = cms.PSet( sigmaIetaIphi = cms.InputTag('electronRegressionValueMapProducer:eleFull5x5SigmaIEtaIPhi'),
```

Comment:
Indeed this requires a ton of variables, consolidation would be a good idea.
Will fix this pending the timeline to the GT (if GT is more than 2 days
away I'll do this here).
Otherwise queue for RECO changes mentioned previously.

On Mon, Aug 17, 2015 at 1:52 PM, Slava Krutelyov <notifications@github.com>
wrote:

> In RecoEgamma/EgammaTools/python/egammaObjectModificationsInMiniAOD_cff.py
> <https://github.com/cms-sw/cmssw/pull/10689#discussion_r37180064>:
>
> > @@ -59,3 +59,68 @@ def setup_mva(val_pset,cat_pset,prod_name,mva_name):
> >            egamma_modifications[3].photon_config,
> >            pho_mva_prod_name,
> >            pho_spring15_50_nt.mvaSpring15NonTrigClassName+pho_spring15_50_nt.mvaTag)
> > +
> > +#############################################################
> > +# REGRESSION MODIFIERS
> > +#############################################################
> > +
> > +egamma_modifications.append(
> > +    cms.PSet( modifierName    = cms.string('EGExtraInfoModifierFromDB'),
> > +              autoDetectBunchSpacing = cms.bool(True),
> > +              bunchSpacingTag = cms.InputTag("addPileupInfo:bunchSpacing"),
> > +              manualBunchSpacing = cms.int32(50),
> > +              rhoCollection = cms.InputTag("fixedGridRhoFastjetAll"),
> > +              vertexCollection = cms.InputTag("offlinePrimaryVertices"),
> > +              electron_config = cms.PSet( sigmaIetaIphi = cms.InputTag('electronRegressionValueMapProducer:eleFull5x5SigmaIEtaIPhi'),
>
> wouldn't it make more sense for the electronRegressionValueMapProducer to
> produce a struct/class with all these variables?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/10689/files#r37180064>.
>